.github/workflows/ci.yml

```diff
@@ -28,6 +28,5 @@ jobs:
         with:
           python-version: "3.10"
       - run: ./setup.sh
-      - run: rustup component add rustfmt --toolchain nightly
-      - run: rustup component add clippy --toolchain nightly
+      - run: cargo +stable install cargo-audit
```

Comment:
Shouldn't this be in the setup script? We use `cargo audit` in the `run_desktop_test`, not only in CI.

Reply:
Moved. I was thinking about being conservative in setup and cargo global installs. But we already do it anyway.